api/swagger.yml

```
@@ -923,6 +923,14 @@ components:
             with an external auth service.
           type: string
           enum: [none, simplified, external]
+        username_placeholder:
```

nit: (none-blocker!)

```suggestion
        username_placeholder_label:
```